apicli/version.py

```diff
@@ -0,0 +1,5 @@
+from distutils.version import StrictVersion
+
+__all__ = ['VERSION']
+
+VERSION = StrictVersion('0.0.1')
```

Comment:
it is __version__ on `setup.py`, there is no real standard for this, every package does it differently, i'm happy to change if you want